### PR TITLE
bump up dependencies

### PR DESCRIPTION
**Pull request by anzioka**

### CODE REVIEW

File: config/webpack/webpack.config.base.js [hunk 1]
```diff
@@ -132,6 +128,12 @@ module.exports = (env) => {
         APP_PATHS.ABS.SOURCE,
         APP_PATHS.ABS.NODE,
       ],
+      fallback: {
+        http: false,
+        zlib: false,
+        https: false,
+        stream: false
+      }
```
Comment (UnsungHero97) [inline config/webpack/webpack.config.base.js]: I don't think these are needed. I did not have to do this for the other apps.

File: package.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@babel/polyfill": "~7.12.1",
```
Comment (UnsungHero97) [inline package.json]: `@babel/polyfill` can be removed.

File: package.json [hunk 3]
```diff
@@ -34,88 +34,88 @@
...
-    "@babel/plugin-transform-runtime": "~7.13.9",
-    "@babel/preset-env": "~7.13.9",
-    "@babel/preset-flow": "~7.12.13",
-    "@babel/preset-react": "~7.12.13",
-    "@babel/register": "~7.13.8",
+    "@babel/plugin-transform-runtime": "~7.13.15",
+    "@babel/preset-env": "~7.14.0",
+    "@babel/preset-flow": "~7.13.13",
+    "@babel/preset-react": "~7.13.13",
+    "@babel/register": "~7.13.16",
```
Comment (UnsungHero97) [inline package.json]: `@babel/register` can be removed.

File: package.json [hunk 4]
```diff
@@ -34,88 +34,88 @@
...
     "babel-loader": "~8.2.1",
     "babel-plugin-styled-components": "~1.12.0",
     "babel-plugin-transform-imports": "~2.0.0",
     "enzyme": "~3.11.0",
     "enzyme-adapter-react-16": "~1.15.6",
-    "eslint": "~7.21.0",
+    "eslint": "~7.25.0",
     "eslint-config-airbnb": "~18.2.1",
     "eslint-import-resolver-webpack": "~0.13.0",
     "eslint-loader": "~4.0.0",
```
Comment (UnsungHero97) [inline package.json]: `eslint-loader` can be removed.

File: package.json [hunk 5]
```diff
@@ -34,88 +34,88 @@
...
     "react-router-dom": "~5.2.0",
-    "redux": "~4.0.0",
+    "redux": "~4.1.0",
     "redux-immutable": "4.0.0",
-    "redux-reqseq": "0.5.0",
+    "redux-reqseq": "~0.6.0",
     "reselect": "~4.0.0",
     "rrule": "~2.6.0",
-    "styled-components": "~5.2.1",
+    "styled-components": "~5.2.3",
```
Comment (UnsungHero97) [inline package.json]: 5.3 was released recently.

File: config/webpack/webpack.config.prod.js [hunk 6]
```diff
@@ -10,8 +10,8 @@ module.exports = (env) => {
   const baseConfig = baseWebpackConfig(env);
 
   const output = Object.assign({}, baseConfig.output, {
-    filename: `${APP_PATHS.REL.STATIC_JS}/app.[hash:8].js`,
-    chunkFilename: `${APP_PATHS.REL.STATIC_JS}/app.chunk.[id].[chunkhash:8].js`,
+    filename: `${APP_PATHS.REL.STATIC_JS}/app.[contenthash].js`,
+    chunkFilename: `${APP_PATHS.REL.STATIC_JS}/app.chunk.[id].[chunkhash].js`,
```
Comment (UnsungHero97) [inline config/webpack/webpack.config.prod.js]: You can remove `inject: true` below.

**Review by UnsungHero97 — CHANGES_REQUESTED** [review]
You're missing `babel.config.js` changes.

File: config/webpack/webpack.config.base.js (hunk 1 above)
Comment (anzioka) [reply config/webpack/webpack.config.base.js]: Without these build fails:

Screen Shot 2021-05-11 at 9 05 30 AM

**Comment by anzioka** [comment]
> You're missing `babel.config.js` changes.

Updated

File: config/webpack/webpack.config.base.js (hunk 1 above)
Comment (UnsungHero97) [reply config/webpack/webpack.config.base.js]: I'm quite certain you don't need the fallbacks. I'll take a look tomorrow.

File: config/babel/babel.config.js [hunk 7]
```diff
@@ -33,8 +33,13 @@ module.exports = {
     }]
   ],
   presets: [
-    '@babel/preset-env',
+    ['@babel/preset-env', {
+      corejs: '3.10',
```
Comment (UnsungHero97) [inline config/babel/babel.config.js]: You need to add `core-js` as a dev dependency in `package.json`. Also, it's on version 3.12 now.

**Review by UnsungHero97 — CHANGES_REQUESTED** [review]

File: config/babel/babel.config.js [hunk 8]
```diff
@@ -34,7 +34,7 @@ module.exports = {
   ],
   presets: [
     ['@babel/preset-env', {
-      corejs: '3.10',
+      corejs: '3.12.1',
```
Comment (UnsungHero97) [inline config/babel/babel.config.js]: https://babeljs.io/docs/en/babel-preset-env#corejs

```suggestion
      corejs: '3.12',
```

**Review by UnsungHero97 — CHANGES_REQUESTED** [review]

**Review by UnsungHero97 — DISMISSED** [review]

**Review by solomontang — APPROVED** [review]

**Review by giuliacampana — APPROVED** [review]

**Review by UnsungHero97 — APPROVED** [review]

**Pull request merged**